client/src/partials/exchange/modal.html

```diff
@@ -10,11 +10,15 @@
 
     <div class="form-group">
       <label>{{ "EXCHANGE.ENTERPRISE_CURRENCY" | translate }}</label>
-      <p class="form-control-static">{{ ModalCtrl.enterpriseCurrency }} : {{ 1 |  currency:ModalCtrl.enterpriseCurrencyId }}</p>
+      <p class="form-control-static">
+        {{ ModalCtrl.enterpriseCurrency }} : {{ 1 |  currency:ModalCtrl.enterpriseCurrencyId }}
```

Comment:
This is another hardcoded value - I haven't looked at this file as a whole so these may well be valid implementation I just can't see exactly what this is doing.